resources/views/contact.blade.php

```diff
@@ -1,5 +1,216 @@
+                        <span class="sr-only">Clutch</span>
+                        <x-icons.clutch class="align-middle" />
+                    </a>
+
+                    <a href="https://github.com/blumilksoftware" class="text-gray-700" target="_blank">
+                        <span class="sr-only">Github</span>
+                        <i class="fa-brands fa-square-github text-xl lg:text-2xl"></i>
+                    </a>
+                    <a href="https://linkedin.com/company/blumilksoftware" class="text-gray-700" target="_blank">
+                        <span class="sr-only">LinkedIn</span>
+                        <i class="fa-brands fa-linkedin text-xl lg:text-2xl"></i>
+                    </a>
+                    <a href="https://www.youtube.com/@blumilksoftware" class="text-gray-700" target="_blank">
+                        <span class="sr-only">Youtube</span>
+                        <i class="fa-brands fa-square-youtube text-xl lg:text-2xl"></i>
+                    </a>
+                    <a href="https://www.facebook.com/blumilksoftware/" class="text-gray-700" target="_blank">
+                        <span class="sr-only">Facebook</span>
+                        <i class="fa-brands fa-square-facebook text-xl lg:text-2xl"></i>
+                    </a>
+                </div>
+            </div>
+        </section>
+    </div>
+    <div id="map" class=" h-[40vh] xl:h-[60vh] w-full"></div>
+    <div class="flex justify-center  space-x-6 md:order-2 pt-8 items-center xl:mt-8 xl:pr-2 xl:hidden pb-6">
+        <a href="https://clutch.co/profile/blumilk-0" class="text-gray-700 align-center fill-current" target="_blank">
+            <span class="sr-only">Clutch</span>
+            <x-icons.clutch class="align-middle" />
+        </a>
+
+        <a href="https://github.com/blumilksoftware" class="text-gray-700" target="_blank">
+            <span class="sr-only">Github</span>
+            <i class="fa-brands fa-square-github text-xl lg:text-2xl"></i>
+        </a>
+        <a href="https://linkedin.com/company/blumilksoftware" class="text-gray-700" target="_blank">
+            <span class="sr-only">LinkedIn</span>
+            <i class="fa-brands fa-linkedin text-xl lg:text-2xl"></i>
+        </a>
+        <a href="https://www.youtube.com/@blumilksoftware" class="text-gray-700" target="_blank">
+            <span class="sr-only">Youtube</span>
+            <i class="fa-brands fa-square-youtube text-xl lg:text-2xl"></i>
+        </a>
+        <a href="https://www.facebook.com/blumilksoftware/" class="text-gray-700" target="_blank">
+            <span class="sr-only">Facebook</span>
+            <i class="fa-brands fa-square-facebook text-xl lg:text-2xl"></i>
+        </a>
+    </div>
+    <script>
+        document.addEventListener('DOMContentLoaded', function() {
+            mapboxgl.accessToken =
+                'pk.eyJ1Ijoia3Jld2FrIiwiYSI6ImNramhmNWI3NjQ2Mm0yeW5xaHUxbmFvMTAifQ.pML1FJHSbrdqOGUvlnd1xw'
```

Comment:
What do you think about putting thus into config file?

Reply:
Or `.env` file

Reply:
During testing I totally forgot about the token. I moved it to .env file.